lib/model/EcoscoreAdjustments.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:json_annotation/json_annotation.dart';
+import '../interface/JsonObject.dart';
+import 'IngredientsOrigins.dart';
+import 'Packaging.dart';
+
+part 'EcoscoreAdjustments.g.dart';
+
+@JsonSerializable()
+class EcoscoreAdjustments extends JsonObject {
+  @JsonKey(includeIfNull: false)
+  Packaging packaging;
+  @JsonKey(name: "origins_of_ingredients", includeIfNull: false)
+  IngredientsOrigins ingredientsOrigins;
```

Comment:
Can we keep the field names the same as in the API? e.g. OriginsOfIngredients instead of IngredientsOrigins
It's easier if the field names are the same as other places on OFF.

Reply:
ok, no problem